gost28147_modes.c

```diff
@@ -282,7 +282,7 @@ static int gost28147_cfb_decrypt_inplace(struct skcipher_walk *walk,
 		src += bsize;
 	} while ((nbytes -= bsize) >= bsize);
 
-	memcpy(walk->iv, iv, bsize);
+	memmove(walk->iv, iv, bsize);
```

Comment:
Hmm. Mainline linux CFB code doesn't contain this memcpy. I should probably drop it alltogether. Would you mind backporting  6c2e322b3621dc8be72e5c86d4fdb587434ba625 instead?